### ISSUE TITLE
Vitest setupFiles not found in src directory

### ISSUE THREAD

**Issue by lasseklovstad** [webpro/knip#233]
Hi! When running `npx knip` it constantly reported Unused file: src/setupTests.ts even though my vitest setup looks like this:
```
export default defineConfig({
    plugins: [react(), svgrPlugin(), eslint(), checker({ typescript: !process.env.VITEST })],
    build: {
        outDir: "build",
    },
    server: {
        port: 3000,
    },
    resolve: {
        alias: [{ find: "@", replacement: path.resolve(__dirname, "src") }],
    },
    test: {
        dir: "src",
        globals: true,
        environment: "jsdom",
        setupFiles: "./src/setupTests.ts",
        reporters: ["junit", "verbose"],
        coverage: {
            reporter: ["cobertura"],
        },
        outputFile: { junit: "junit.xml" },
        testTimeout: 20000,
    },
})
```

After moving setupFile.ts to a folder inside the src directory and adjusted the vitest config to reflect it, knip detected the file.
I moved it to src/test/setupTest.ts and the change in config:
```
export default defineConfig({
    test: {
        setupFiles: "./src/test/setupTests.ts",
    },
})
```

**Comment by webpro** [comment]
Sounds odd. Is the `dir` relevant here? Have a rep(r)o I can look at?

**Comment by lasseklovstad** [comment]
https://stackblitz.com/edit/vitejs-vite-f3k8tj?file=package.json

Run `npx knip`

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.24.0. See [Release 2.24.0](https://github.com/webpro/knip/releases/tag/2.24.0) for release notes._

**Comment by webpro** [comment]
Turned out the vite plugin didn't mimic the vitest plugin magic yet. Thanks for reporting, Lasse!
